test/Lachain.CoreTest/IntegrationTests/VirtualMachineTest.cs

```diff
@@ -60,8 +60,8 @@ public void Teardown()
          //   _container.Dispose();
         }
 
-        /*
-        [Test]
+        
+        /*[Test]
```

Comment:
Why this test is commented in this pull request? Do we have a test case for storage string? VirtualMachineTests are really helpful at higher levels testing.)